stubs/Markdown/markdown/blockprocessors.pyi

```diff
@@ -53,7 +59,7 @@ class SetextHeaderProcessor(BlockProcessor):
 class HRProcessor(BlockProcessor):
     RE: str = ...
     SEARCH_RE: Pattern[str]
-    match: Any
+    match: Match[str] # TODO: This type won't be compatible in 3.10.
```

Comment:
Can you clarify this? If you're referring to `typing.Match` vs. `re.Match`, either will work in stubs.

Reply:
I'm pretty sure `match` becomes a keyword in 3.10. Not sure how that affects stubs.

Reply:
The new PEG parser in 3.9 allows for "soft" keywords, so you can still use match as a variable name or whatever

Reply:
Oh, really? That's very interesting to hear. I haven't really upgraded my Python above 3.8 so I haven't looked into all of the big differences after.

Reply:
A bit off-topic, but the soft keywords lead to weird tricks that you can do (don't try at home, lol):

```python
>>> case = print
>>> match = print
>>> match match:
...     case case:
...             match(match(case(case)))
... 
<built-in function print>
None
None
>>> 
```